init.vim

```diff
@@ -140,7 +128,74 @@ nnoremap <silent> <Left> :vertical resize -5<cr>
 nnoremap <silent> <Up> :resize +5<cr>
 nnoremap <silent> <Down> :resize -5<cr>
 " === END Resize panes ===
-
+"
+" ============================================================
+" === Telescope mapings
+" ============================================================
+nnoremap <leader>ff <cmd>lua require('telescope.builtin').find_files()<cr>
+nnoremap <leader>fg <cmd>lua require('telescope.builtin').live_grep()<cr>
+nnoremap <leader>fb <cmd>lua require('telescope.builtin').buffers()<cr>
+nnoremap <leader>fh <cmd>lua require('telescope.builtin').help_tags()<cr>
+" ============================================================
+" === END Telescope mapings
+" ============================================================
+"
+"=============================================================
+" === Compe completion
+" === - https://github.com/hrsh7th/nvim-compe
+"=============================================================
+inoremap <silent><expr> <C-Space> compe#complete()
+inoremap <silent><expr> <CR>      compe#confirm('<CR>')
+inoremap <silent><expr> <C-e>     compe#close('<C-e>')
+inoremap <silent><expr> <C-f>     compe#scroll({ 'delta': +4 })
+inoremap <silent><expr> <C-d>     compe#scroll({ 'delta': -4 })
+"=============================================================
+" === END Compe
+"=============================================================
+"
+"=============================================================
+" === LSPSAGA
+" === - https://github.com/glepnir/lspsaga.nvim
+"=============================================================
+nnoremap <silent> gh <cmd>lua require'lspsaga.provider'.lsp_finder()<CR>
```

Comment:
This line requires the following comment above it:

```
-- lsp provider to find the cursor word definition and reference
```

Reply:
Techdebt ticket raised https://github.com/js-jslog/dotfiles/pull/7#discussion_r607311492